Documentation/gettingstarted_kafka.rst

```diff
@@ -0,0 +1,411 @@
+
+::
+
+  apiVersion: "cilium.io/v2"
+  kind: CiliumNetworkPolicy
+  description: "enable outposts to consume empire-announce"
+  metadata:
+    name: "rule2"
+  spec:
+    endpointSelector:
+      matchLabels:
+        app: kafka
+    ingress:
+    - fromEndpoints:
+      - matchLabels:
+          app: empire-outpost
+      toPorts:
+      - ports:
+        - port: "9092"
+          protocol: TCP
+        rules:
+          kafka:
+          - apiKey: "fetch"
+            topic: "empire-announce"
+          - apiKey: "apiversions"
+          - apiKey: "metadata"
+          - apiKey: "findcoordinator"
+          - apiKey: "joingroup"
+          - apiKey: "leavegroup"
+          - apiKey: "syncgroup"
+          - apiKey: "offsets"
+          - apiKey: "offsetcommit"
+          - apiKey: "offsetfetch"
+          - apiKey: "heartbeat"
+
+
+CiliumNetworkPolicies operate by white-listing allowed traffic, meaning that requests that do not match
+any rules are denied as invalid.
+
+The above rule applies to inbound (i.e., "ingress") connections to kafka-broker pods (as
+indicated by “app: kafka”
+in the "endpointSelector" section).  The rule will apply to connections from pods with label
+"app: empire-outpost" as indicated by the "fromEndpoints" section.   The rule explicitly matches
+Kafka connections destined to TCP 9092, and white-lists a list of protocol requests according to various
+fields found in Kafka protocol messages.  Specifically, each request contains an "apiKey" field that
+indicates the type of request, and we white-list several apiKey's here, the most important being the
+apiKey "fetch",
+which allows the client to consume from a particular topic.   Notice that requests with "apiKey: fetch"
+are further limited by a "topic" field, in this case "empire-announce".
+
+The full policy add two additional rules that permit the legitimate produce request
+(topic "empire-announce" and topic "deathstar-plans") from *empire-hq* and the
+legitimate fetch requests (topic = "deathstar-plans") from *empire-backup*.  The full policy
+can be reviewed by opening the URL in the command below in a browser.
+
+Apply this Kafka-aware network security policy using kubectl in the main window:
+
+.. parsed-literal::
+
+    $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes-kafka/kafka-sw-security-policy.yaml
+
+If we then again try to produce a message from outpost-9999 to “empire-annnounce”, it is denied.
+Type control-c and then run:
+
+::
+
+  $ echo “Vader Trips on His Own Cape” | ./kafka-produce.sh --topic empire-announce
+  [2017-10-31 07:08:34,088] ERROR Error when sending message to topic empire-announce with key: null, value: 33  bytes with error: (org.apache.kafka.clients.producer.internals.ErrorLoggingCallback)
+  org.apache.kafka.common.errors.TopicAuthorizationException: Not authorized to access topics: [empire-announce]
+
+This is because the policy does not allow messages with apiKey = “produce” for topic “empire-announce” from
```

Comment:
fix invalid formatting/ characters

Reply:
â  âproduceâ for topic âempire-announceâ